client-app/src/styles/default.scss

```diff
@@ -1,2 +1,3 @@
 @import 'bootstrap/scss/bootstrap';
-// import you custom styles here
\ No newline at end of file
+@import 'bootstrap-vue/src/index.scss';
```

Comment:
I don't like to include all bootstrap-vue scss in case we use only bootstrap-vue table. Replace (and test please) it with the following code:
```scss
@import 'bootstrap-vue/src/variables';
@import 'bootstrap-vue/src/utilities';
@import 'bootstrap-vue/src/components/table/index';
```

Reply:
I propose to do this later. If we will have problems with it.